corpus/corpus/settings.py

```diff
@@ -40,6 +40,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "pages.apps.PagesConfig",
+    "config",
```

Comment:
Add the `config` app notation.
```suggestion
    "config.apps.ConfigConfig",
```